lib/shopify-cli/admin_api.rb

```diff
@@ -134,8 +134,12 @@ def fetch_api_version(ctx, api_version, shop)
         )
         CLI::Kit::Util.begin do
           versions = client.query("api_versions")["data"]["publicApiVersions"]
-          latest = versions.find { |version| version["displayName"].include?("Latest") }
-          latest["handle"]
+          latest = versions
+            .select { |version| version["supported"] }
+            .map { |version| version["handle"] }
+            .sort
+            .reverse[0]
+          latest
```

Comment:
Nit: we could simplify this a bit

```suggestion
          versions
            .select { |version| version["supported"] }
            .map { |version| version["handle"] }
            .sort
            .reverse[0]
```